sys-kernel/vanilla-kernel/vanilla-kernel-5.4.1.ebuild

```diff
@@ -0,0 +1,165 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit mount-boot savedconfig toolchain-funcs
+
+MY_P=linux-${PV}
+DESCRIPTION="Linux kernel built from vanilla upstream sources"
+HOMEPAGE="https://www.kernel.org/"
+SRC_URI="https://cdn.kernel.org/pub/linux/kernel/v5.x/${MY_P}.tar.xz"
+S=${WORKDIR}/${MY_P}
+
+LICENSE="GPL-2"
+SLOT="${PV}"
+KEYWORDS="~amd64"
+IUSE="+initramfs"
+
+# install-DEPEND actually
+RDEPEND="sys-apps/debianutils
+	initramfs? ( sys-kernel/dracut )"
```

Comment:
genkernel can also generate initramfs, so we can provide it as an alternative here

Reply:
genkernel is horrible, so I'd rather not waste my time on figuring it out.